examples/model_service_template/mxnet_vision_batching.py

```diff
@@ -0,0 +1,219 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import mxnet as mx
+import json
+import os
+import numpy as np
+from collections import namedtuple
+import logging
+
+
+class MXNetVisionServiceBatching(object):
+    def __init__(self):
+        """
+        Initialization for MXNet Vision Service supporting batch inference
+        """
+        self.mxnet_ctx = None
+        self.mx_model = None
+        self.labels = None
+        self.signature = None
+        self.epoch = 0
+        self.error = None
+        self._context = None
+        self._batch_size = 0
+        self.initialized = False
+        self.erroneous_reqs = set()
+
+    def top_probability(self, data, labels, top=5):
```

Comment:
this can be static or suppress pycharm warning with: # noinspection PyMethodMayBeStatic